content/en/integrations/guide/pivotal-cloud-foundry-manual-setup.md

```diff
@@ -0,0 +1,502 @@
+---
+title: Pivotal Cloud Foundry Manual Setup Guide
+kind: guide
+description: "Steps for manually setting up the Pivotal Cloud Foundry Integration"
+aliases:
+  - /integrations/pivotal-platform
```

Comment:
The alias for the page already exists in `content/en/integrations/vmware_tanzu_application_service.md`.

```suggestion
```

Reply:
Thanks for catching this!